src/OpenTelemetry.Contrib.Instrumentation.Wcf/README.md

```diff
@@ -20,6 +20,24 @@ using var openTelemetry = Sdk.CreateTracerProviderBuilder()
     .Build();
 ```
 
+Instrumentation can be configured via options overload for `AddWcfInstrumentation` method:
+
+```csharp
+using var openTelemetry = Sdk.CreateTracerProviderBuilder()
+    .AddWcfInstrumentation(options => 
+    {
+        options.SuppressDownstreamInstrumentation = false;  
+        options.Enrich = (activity,eventName,message) =>    // -> Enable enriching an activity after it is created.
+        {
+            if(eventName == "AfterReceiveRequest")  // For event names, please refer to string contstants in WcfEventNames class.
```

Comment:
Might be nice to use `WcfEventNames` in the example so people know it is available.

Reply:
Absolutely.  Done.